### DISCUSSION TITLE
How to show loading state when a #[Computed] property is doing it's magic?

### DISCUSSION THREAD

**Discussion by adrianlzx1996** [livewire/livewire#7959] [1. Help] [answered]
This use case would be having a filter form and a table with list of records where the filtered query will take some time to finish.

While the query is running, i would like to show a loading icon or animation to indicate the loading state.

Does anyone has any idea of this?

**Comment by danie-ramdhani** [comment]
use `wire:loading` (https://livewire.laravel.com/docs/wire-loading#basic-usage). here's example:

```blade
    <form wire:submit="filter">
        {{-- ... --}}

        <button type="submit">filter!</button>

        <div wire:loading wire:target="filter">
            {{-- put the loading indicator here --}}
        </div>
    </form>
```

**Reply by adrianlzx1996** [reply]
Hey @danie-ramdhani 

Thank you for your suggestion! 

However, what if i'm using LIVE filter instead of a Filter button?

**Reply by danie-ramdhani** [reply, marked as answer]
```blade
    <input type="text" wire:model.live="filterProp">

    <div wire:target="filterProp" wire:loading>
        <span>Filtering..</span>
    </div>
```

**Reply by adrianlzx1996** [reply]
Hey @danie-ramdhani 

Thank you so much, this works even if i have multiple filter field!
